src/Platforms/SqlitePlatform.php

```diff
@@ -347,7 +347,6 @@ public function getForeignKeyDeclarationSQL(ForeignKeyConstraint $foreignKey)
      */
     protected function _getCreateTableSQL($name, array $columns, array $options = [])
     {
-        $name        = str_replace('.', '__', $name);
```

Comment:
added by https://github.com/doctrine/dbal/commit/679854845c92b8d651fe4a3c310d44e3dcca2f46

related issue https://github.com/doctrine/orm/issues/2132 - "eat prefixed tables" hack is wrong and with proper schema/attach support it is not needed